libs/SalesforceHybridSDK/SalesforceHybridSDK/Classes/SFHybridViewController.m

```diff
@@ -63,6 +64,8 @@ @interface SFHybridViewController()
     SFHybridViewConfig *_hybridViewConfig;
 }
 
+@property (nonatomic, readwrite, assign) BOOL useWKWebView;
```

Comment:
Fair enough.  If we were keeping both around for any length of time, a lot of the conditionalizing otherwise feels like subclasses implementing their specific logic.  I'm cool with that approach, though. 👍